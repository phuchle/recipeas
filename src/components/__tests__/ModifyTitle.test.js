import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ModifyTitle from '../ModifyTitle';

const props = {
  nextButton: true,
  plusButton: true,
  ingredientList: [],
  modifyTempTitle: jest.fn()
};

describe('ModifyTitle', () => {
  test('renders without crashing', () => {
    const component = shallow(<ModifyTitle {...props} />);
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});
