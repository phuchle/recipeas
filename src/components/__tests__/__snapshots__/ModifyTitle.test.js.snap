// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModifyTitle renders without crashing 1`] = `
<div>
  <h4>
    Recipe Information
  </h4>
  <FormGroup
    bsClass="form-group"
    controlId="recipeTitle"
    validationState="error"
  >
    <ControlLabel
      bsClass="control-label"
      srOnly={false}
    >
      Title
    </ControlLabel>
    <FormControl
      bsClass="form-control"
      componentClass="input"
      onChange={[Function]}
      placeholder="Enter an awesome title"
      type="text"
      value=""
    />
    <FormControlFeedback
      bsClass="form-control-feedback"
      bsRole="feedback"
    />
    <HelpBlock
      bsClass="help-block"
    >
      A title is required.
    </HelpBlock>
  </FormGroup>
  <ControlLabel
    bsClass="control-label"
    srOnly={false}
  >
    Servings
  </ControlLabel>
  <FormControl
    bsClass="form-control"
    componentClass="input"
    onChange={[Function]}
    placeholder="How many people will this recipe feed?"
    type="text"
    value=""
  />
  <ControlLabel
    bsClass="control-label"
    srOnly={false}
  >
    Allergens
  </ControlLabel>
  <FormControl
    bsClass="form-control"
    componentClass="input"
    onChange={[Function]}
    placeholder="What allergens does this food contain?"
    type="text"
    value=""
  />
  <form>
    <Button
      active={false}
      block={true}
      bsClass="btn"
      bsSize="large"
      bsStyle="primary"
      disabled={true}
      onClick={[Function]}
      type="submit"
    >
      Next
    </Button>
    <Link
      replace={false}
      to="/"
    >
      <Button
        active={false}
        block={true}
        bsClass="btn"
        bsSize="large"
        bsStyle="default"
        disabled={false}
        onClick={[Function]}
      >
        Back
      </Button>
    </Link>
  </form>
</div>
`;
